language/move-lang/tests/move_check/typing/constant_unsupported_exps.exp

```diff
@@ -66,261 +66,288 @@ error:
 
     ┌── tests/move_check/typing/constant_unsupported_exps.move:14:9 ───
     │
- 14 │         foo();
-    │         ^^^^^ Module calls are not supported in constants
+ 14 │         f_public();
+    │         ^^^^^^^^^^ Module calls are not supported in constants
     │
 
 error: 
 
     ┌── tests/move_check/typing/constant_unsupported_exps.move:15:9 ───
     │
- 15 │         borrow_global<R>(0x42);
-    │         ^^^^^^^^^^^^^^^^^^^^^^ 'borrow_global' is not supported in constants
+ 15 │         f_script();
+    │         ^^^^^^^^^^ Invalid call to '0x42::M::f_script'
+    ·
+ 39 │     public(script) fun f_script() {}
+    │                        -------- This function can only be called from a script context, i.e. a 'script' function or a 'public(script)' function
```

Comment:
I am a bit concerned that both this error and the `module calls are not supported in constants` are shown here, while for the rest of the unsupported operations in this file, it is only `<some_operation> are not supported in constants`.

Reply:
It should be fine and is the expected behavior (we don't stop on the first error). You should get a double error for calling a private function too (from another module)